test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -762,6 +762,84 @@ def check_proto():
 
         self.assertRaises(RuntimeError, check_proto)
 
+    def test_is_fp_for_C_TypeList(self):
```

Comment:
I'm thinking maybe the test case model with context is great for `test_models_onnxruntime.py`, where knowledge of model is kept through exact source code, and test covers the exact same usage.

For lower level test cases, such as this test case name implies, would it be better to test over a more narrowed downed unit repro case? The benefit would be easier maintainability, and less background context required to understand.